neurolib/models/hopf/loadDefaultParams.py

```diff
@@ -58,21 +56,28 @@ class struct(object):
     # external input parameters:
     params.tau_ou = 5.0  # ms Timescale of the Ornstein-Uhlenbeck noise process
     params.sigma_ou = 0.0  # mV/ms/sqrt(ms) noise intensity
-    params.x_ext_mean = 0.0  # mV/ms (OU process) [0-5]
-    params.y_ext_mean = 0.0  # mV/ms (OU process) [0-5]
+    params.x_ou_mean = 0.0  # mV/ms (OU process) [0-5]
+    params.y_ou_mean = 0.0  # mV/ms (OU process) [0-5]
 
     # neural mass model parameters
     params.a = 0.25  # Hopf bifurcation parameter
     params.w = 0.2  # Oscillator frequency, 32 Hz at w = 0.2
 
     # ------------------------------------------------------------------------
 
-    params.xs_init = 0.05 * np.random.uniform(0, 1, (params.N, 1))
-    params.ys_init = 0.05 * np.random.uniform(0, 1, (params.N, 1))
+    # initial values of the state variables
+    params.xs_init = 0.5 * np.random.uniform(-1, 1, (params.N, 1))
+    params.ys_init = 0.5 * np.random.uniform(-1, 1, (params.N, 1))
+
+    # Ornstein-Uhlenbeck noise state variables
+    params.x_ou = np.zeros((params.N,))
+    params.y_ou = np.zeros((params.N,))
 
-    params_dict = params.__dict__
+    # values of the external inputs
+    params.x_ext = np.zeros((params.N,))
+    params.y_ext = np.zeros((params.N,))
 
-    return params_dict
+    return params
 
 
 def computeDelayMatrix(lengthMat, signalV, segmentLength=1):
```

Comment:
this function is in all `loadDefaultParams.py` files (for each model). we might do some refactoring on this...
also, I would think about parameters, I personally do not like setting parameters inside python files... I am usually loading parameters from `yaml` or `json`... that way, we could provide `default_aln_params.yaml` for example and just do loading function...

Reply:
This one should be refactored, I agree. Not gonna do it for this PR, hope we won't forget about it 👍